geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -5718,7 +5718,6 @@ RegionEntry basicPutEntry(final EntryEventImpl event, final long lastModified)
   @Override
   public long basicPutPart2(EntryEventImpl event, RegionEntry entry, boolean isInitialized,
       long lastModified, boolean clearConflict) {
-
```

Comment:
put this blank line back so that this PR does not modify LocalRegion.java

Reply:
updated.